.github/workflows/examples/edd.yml

```diff
@@ -0,0 +1,115 @@
+name: Easy Digital Downloads Updater
+
+# Run this workflow twice daily
+on:
+  # schedule:
+  #   - cron: '0 */12 * * *'
+  workflow_dispatch:
+
+env:
+  # The plugin slug
+  PACKAGE_SLUG: example
+
+  # The URL of the vendor using EDD (example: https://searchwp.com)
+  VENDOR_URL: https://example.com
+
+  # The item ID of the download, this is different for every plugin and vendor, for example SearchWP's item ID is `216029`
+  ITEM_ID: 1
+
+  # A valid URL that your purchase is registered to in EDD
+  ACTIVATION_URL: ${{github.server_url}}/${{github.repository}}
+
+jobs:
+  update-plugin:
+    name: Fetch and check versions
+    runs-on: ubuntu-latest
+
+    steps:
+
+      # Checkout repo
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # Use EDD API to find download link for latest version, using the EDD item ID (different for every plugin)
+      - name: Get download link from EDD
+        id: EDDResponse
+        uses: fjogeleit/http-request-action@master
+        with:
+          url: '${{ env.VENDOR_URL }}/?edd_action=get_version&item_id=${{ env.PACKAGE_SLUG }}&license=${{secrets.LICENSE_KEY}}&url=${{ env.ACTIVATION_URL }}'
```

Comment:
I believe item_id will accept either the slug or ID, but I would give it the ID - more explicit and more logically matches the param.

Reply:
Alternatively, this example doesn't need both env values and could use only the slug.